cookiecutter.json

```diff
@@ -2,14 +2,16 @@
     "full_name": "Jørgen S. Dokken",
     "email": "dokken@simula.no",
     "github_username": "jorgensd",
-    "project_name": "Package Name",
-    "module_name": "{{ cookiecutter.project_name.lower().replace(' ', '_').replace('-', '_') }}",
+    "project_name": "Research Project 1",
+    "repository_name": "{{ cookiecutter.project_name.lower().replace(' ', '_').replace('-', '_') }}",
+    "module_name": "{{cookiecutter.repository_name}}",
     "project_short_description": "Tools ensuring reproducible and standarized Python packaging.",
     "version": "0.1.0",
     "linting": "y",
     "testing": ["pytest", "pytest-cov", "none"],
```

Comment:
I think it is a bit strange that I have to pick either `pytest` or `pytest-cov` if I want both. A more natural selection would be [`pytest`, `unittest` `none`]

Reply:
Pytest-cov is currently just pytest plus the coverage reports.

it is to make it possible to distinguish between the different workflows in github, as we cant use cookiecutter on them.

as a side note, we do not currently support the `unittest` setup.

Reply:
See #3